### DISCUSSION TITLE
How to remove the symbol on submit

### DISCUSSION THREAD

**Discussion by GarciaBrix** [dm4t2/vue-currency-input#315] [Q&A]
![image](https://user-images.githubusercontent.com/99477124/153888932-bfc587bf-3875-4c7c-9711-20d47dc8a583.png)
How do I removed the symbol on submitting? 

`
<template>
    <currency-input
        v-if="column.type === 'currency'"
        :placeholder="column.placeholder ??  column.label"
        :name="column.key" v-model="column.data"
        :options="{ currency: 'EUR', exportValueAsInteger: true}"
    />
    <input v-else :readonly="column.readonly" :type="column.type" :name="column.key" v-model="column.data" :placeholder="column.placeholder ??  column.label">
</template>

<script lang="ts">
import {defineComponent, PropType} from 'vue';
import {FormDefinition} from "../../../interfaces/components/FormDefinition";
import CurrencyInput from './CurrencyInput.vue'

export default defineComponent({
    components: {CurrencyInput},
    props: {
        column: {
            type: Object as PropType<FormDefinition>,
            required: true,
        },
    },
});
</script>
`

**Comment by dm4t2** [comment]
The example component from the [docs](https://dm4t2.github.io/vue-currency-input/guide.html#usage) shows how to work with plain numbers:

https://codesandbox.io/s/vue-currency-input-vue-3-example-5l51f?from-embed=&file=/src/CurrencyInput.vue
